### DISCUSSION TITLE
`w-full` bug, idk maybe its just me.

### DISCUSSION THREAD

**Discussion by Crepyz** [tailwindlabs/tailwindcss#12820] [Help]
**What component (if applicable)**
- URL for category: none
- Component name: idk

**Describe the bug**
Theres a white space on the right everytime I use `w-full` code.

**Expected behavior**
I expect the width takes 100% of the screen

**Screenshots**
![Screenshot 2024-01-24 193009](https://github.com/tailwindlabs/tailwindui-issues/assets/154492162/c07f3002-93f6-4207-aece-d622a366d5eb)

**Browser/Device (if applicable)**
 - OS: Windows 11 pro
 - Browser: Microsoft Edge
 - Version: Latest

**Additional context**
Nothing.

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) that reproduces the unexpected behavior or a project that does so.

**Comment by LinThitHtwe** [comment]
I think something is overflowing, you could add ```overflow-x-hidden``` to solve this
